res/skins/QMLDemo/FadeBehavior.qml

```diff
@@ -0,0 +1,33 @@
+import QtQuick 2.12
+
+Behavior {
+    id: root
+
+    // Starting with QtQuick 2.15, we can use targetProperty.object here
+    property Item fadeTarget
+
+    SequentialAnimation {
+        NumberAnimation {
+            target: root.fadeTarget
+            property: "opacity"
+            from: 1
+            to: 0
+            easing.type: Easing.InQuad
+            duration: 150
+        }
+
+        PropertyAction {
```

Comment:
I know you've taken this code from some example code on the Behavior Type but I still struggle to understand it...

Reply:
Added some comments.